pkg/cluster/spec/grafana.go

```diff
@@ -201,7 +201,8 @@ func (i *GrafanaInstance) InitConfig(
 func (i *GrafanaInstance) initDashboards(e executor.Executor, spec GrafanaSpec, paths meta.DirPaths, clusterName string) error {
 	dashboardsDir := filepath.Join(paths.Deploy, "dashboards")
 	// To make this step idempotent, we need cleanup old dashboards first
-	cmd := fmt.Sprintf("mkdir -p %[1]s && rm -f %[1]s/*.json", dashboardsDir)
+	// ignore the `not_match` error if files were not found in zsh
+	cmd := fmt.Sprintf("mkdir -p %[1]s && rm -f %[1]s/*.json || exit 0", dashboardsDir)
```

Comment:
What if `mkdir` failed? Maybe we should use find with `-exec`

Reply:
ref https://man7.org/linux/man-pages/man1/mkdir.1.html
> -p, --parents
>              no error if existing, make parent directories as needed

Seems `mkdir -p` will never failed. I'll use `find -delete` to replace with it.